ledger/eval/eval_test.go

```diff
@@ -52,7 +52,7 @@ var testSinkAddr = basics.Address{0x2c, 0x2a, 0x6c, 0xe9, 0xa9, 0xa7, 0xc2, 0x8c
 var minFee basics.MicroAlgos
 
 func init() {
-	params := config.Consensus[protocol.ConsensusCurrentVersion]
+	params := config.Consensus[protocol.ConsensusFuture]
```

Comment:
Is this minFee constant used extensively? Seems like it will bite us if it's used both in one-off tests that test `current` and some that use `future`.

Though I guess you rooted out any that test current. Still, maybe determine the min fee properly in the tests using the proto version in the test?